File: 프로그래머스/0/181888. n개 간격의 원소들/n개 간격의 원소들.js
function solution(num_list, n) {
        var count1=0;
        var m=0;
        while(true){
            if(m*n<num_list.length){ 
                count1++;
                m++;
            }
            else break;
        }
        var answer = new Array(count1);
        var count=0;
        for(var i=0; i<answer.length;i++){
            answer[i]=num_list[count];
            count+=n;
        }
    return answer;
}

File: 프로그래머스/0/181888. n개 간격의 원소들/n개 간격의 원소들.py
def solution(num_list, n):
    count1 = 0
    m = 0
    while True:
        if m*n < len(num_list):
            count1 = count1+1
            m = m+1
        else: break
    answer = [0] * count1
    count=0
    for i in range(len(answer)):
        answer[i]=num_list[count]
        count+=n;
    return answer